File: a2_funcs/fib.py
def fib_recursive(n: int)->int:
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib_recursive(n - 1) + fib_recursive(n - 2)


n = int(input())
print(fib_recursive(n))

File: a2_funcs/fib.cpp
#include <iostream>

int fib_recursive(int n) {
	if (n == 0) {
		return 0;
	} else if (n == 1) {
		return 1;
	} else {
		return fib_recursive(n - 1) + fib_recursive(n - 2);
	}
}

int main() {
	int n;
	std::cin >> n;

	std::cout << fib_recursive(n) << '\n';

	return 0;
}
